src/state/stake/hooks.ts

```diff
@@ -96,6 +95,7 @@ export interface CommonStakingInfo {
   pair: string;
 
   quickPrice: number;
+  dQuickPrice: number;
```

Comment:
The dQuick price and quick price don't really have anything to do with the staking token, there's no reason they should be copied to everyone this should be located in a single place and retrieved only when needed

Reply:
Removed `quickPrice` and `dQuickPrice` from the stakingInfo property